src/test/java/ConsolidatorTest.java

```diff
@@ -40,19 +40,19 @@ public void testSimpleCopyCellsToOtherExcel() {
         ArrayList<Excel> excels = new ArrayList<>();
         excels.add(dst);
 
-        new Consolidator(Properties.getMappingFilePath(), src, excels, 0);
+        new Consolidator(Properties.getMappingFilePath(), src, excels);
         dst.save();
 
         XlsxExcelFile excel = new XlsxExcelFile(TestHelper.getWritableFilePath(), 0);
-        assertEquals(src.getCellValue(0,0), excel.getCellValue(0, 0));
-        assertEquals(src.getCellValue(0,1), excel.getCellValue(0, 1));
-        assertEquals(src.getCellValue(0,2), excel.getCellValue(0, 2));
-        assertEquals(src.getCellValue(1,0), excel.getCellValue(1, 0));
-        assertEquals(src.getCellValue(1,1), excel.getCellValue(1, 1));
-        assertEquals(src.getCellValue(1,2), excel.getCellValue(1, 2));
-        assertEquals(src.getCellValue(2,0), excel.getCellValue(2, 0));
-        assertEquals(src.getCellValue(2,1), excel.getCellValue(2, 1));
-        assertEquals(src.getCellValue(2,2), excel.getCellValue(2, 2));
+        assertEquals(src.getCellValue(0,0).toString(), excel.getCellValue(0, 0).toString());
+        assertEquals(src.getCellValue(0,1).toString(), excel.getCellValue(0, 1).toString());
```

Comment:
Not sure if we really should convert the output to a string. Aren't we tricking here (regarding the returned types)?